cmd/aperturectl/cmd/utils/agents.go

```diff
@@ -4,11 +4,13 @@ import (
 	"context"
 	"fmt"
 
-	"google.golang.org/protobuf/types/known/emptypb"
+	cmdv1 "github.com/fluxninja/aperture/v2/api/gen/proto/go/aperture/cmd/v1"
 )
 
-func ListAgents(client IntrospectionClient) error {
-	agents, err := client.ListAgents(context.Background(), &emptypb.Empty{})
+func ListAgents(client IntrospectionClient, agentGroup string) error {
```

Comment:
Could you please change the cloud agents to also use this method?

Reply:
I've changed it [here](https://github.com/fluxninja/aperture/blob/1a15d9e6bcb21428a6474a42deeccdac8e46d1eb/cmd/aperturectl/cmd/cloud/agents.go#L33) if that's what you meant.

Reply:
No I meant use this method from utils/agents.go instead of having same code at 2 places

Reply:
Modified.